### PR TITLE
Show link to etherscan for the transaction hash

### PR DESCRIPTION
**Pull request by agatsoh**
Shows a link to Etherscan while transferring ETH to the raiden account

### CODE REVIEW

**Comment by agatsoh** [comment]
@fredo hope you could review this one.

File: raiden_installer/web.py [hunk 1]
```diff
@@ -121,6 +121,11 @@ def _send_summary(self, text, **kw):
             message["icon"] = icon
         self.write_message(message)
 
+    def _send_txhash_message(self, text, **kw):
```
Comment (fredo) [inline raiden_installer/web.py:124]: Just a nitpick, if you already go that way and build a function to send a message with tx_hash property, why do you go the way of passing it through `kw`
Couldnt you just create an tx_hash argument?

Comment (agatsoh) [reply raiden_installer/web.py:124]: Yes thats also a good point, if we need something more then probably we can pass it later as well. Right now just keeping it `tx_hash` will increase the readable.

**Comment by agatsoh** [comment]
Done the changes @fredo you can check again.

**Pull request merged**